environment/Dockerfile

```diff
@@ -14,27 +14,27 @@ FROM ubuntu:22.04
 # openocd: Our embedded GDB server of choice
 # usbutils: USB tools (lsusb)
 # nano: Basic Unix text editor
+# libgtk-3-0: Random library required for running STM32CubeMX in container
+# libxi6: Random library required for running STM32CubeMX V6 in container
+# libxtst6: Random library required for running STM32CubeMX V6 in container
+# libxrender1: Random library required for running STM32CubeMX V6 in container
 RUN apt update -y && apt install -y \
     wget \
     unzip \
     pip \
     git \
     git-lfs \
```

Comment:
if we clone from outside and volume, perhaps this is not used either. (git, gitlfs, unzip, wget??)

Reply:
if I remember correctly I did have to install these to get the CI working but that was a long time ago
container takes too long to build for me to want to try